minio/minio
# fix: Incorrect ServersCount in cluster.info

## Description

The `ServersCount` field in cluster.info is expected to contain the
number of nodes, and not number of endpoints.

Fixed by using length of `globalNotificationSys.ServerInfo()` and adding
one for the local node.

## Motivation and Context

Bugfix

## How to test this PR?

- Generate a profile using `mc support profile`
- Extract the generated zip file and inspect contents of `cluster.info`
- Verify that in case of a single server setup with multiple endpoints, the value of `servers_count` is `1`

## Types of changes

## Checklist:
